internal/command/run_create.go

```diff
@@ -19,12 +19,29 @@ type CreateRunCommand struct {
 	Workspace              string
 	ConfigurationVersionID string
 	Message                string
+	TargetAddrs            []string
 
 	PlanOnly  bool
 	IsDestroy bool
 	SavePlan  bool
 }
 
+// flagStringSlice is a flag.Value implementation which allows collecting
+// multiple instances of a single flag into a slice. This is used for flags
+// such as -target=aws_instance.foo and -var x=y.
+type flagStringSlice []string
+
+var _ flag.Value = (*flagStringSlice)(nil)
+
+func (v *flagStringSlice) String() string {
+	return ""
```

Comment:
I am wondering if instead of a blank string, we should be returning a string representation of the value of flag?
```suggestion
// import strings
return strings.Join(*v, ",")
```

Reply:
Great idea. I copied these functions from the Terraform core repository and how it handles the flags, but I like this approach.